pkg/autoscaler/statforwarder/forwarder_test.go

```diff
@@ -439,18 +443,18 @@ func TestProcess(t *testing.T) {
 		t.Fatalf("Timeout waiting f.processors got updated")
 	}
 
-	// Stat1 should be accepted and stat2 should be forwarded.
-	f.Process(stat1)
-	f.Process(stat2)
-
-	if got, want := acceptCount, 1; got != want {
-		t.Errorf("acceptCount = %d, want = %d", got, want)
+	// Wait for the stat enqueued previously to be retried.
+	got := <-acceptCh
+	if got != 1 {
+		t.Fatalf("Want acceptCount = 1, got %v", got)
 	}
 
-	// One more accept.
+	// Once more acception.
```

Comment:
I don't think `acception` is a word, maybe `Accept once more` or `Accept again`?